README.md

```diff
@@ -58,6 +67,53 @@ Below are sample usages for the playbook:
       force_reactivation: false
 ```
 
+### Download from this repo
```

Comment:
Sorry I didn't catch this but I think this should say 'Clone from this repo' so that it matches the wording at the top.